docs/getting-started.md

```diff
@@ -38,6 +38,19 @@ alias butane='podman run --rm --tty --interactive \
               quay.io/coreos/butane:release'
 ```
 
+Alternatively you could also create a wrapper script at `~/.local/bin/butane`:
+
+```bash
+#!/bin/sh
+exec podman run --rm --tty --interactive \
+     --security-opt label=disable        \
+     --volume "$PWD":/pwd --workdir /pwd \
+     quay.io/coreos/butane:release       \
+     "$@"
```

Comment:
```suggestion
     --volume "${PWD}":/pwd --workdir /pwd \
     quay.io/coreos/butane:release       \
     "${@}"
```

Also minor. I'm wondering why we do not have quotes for the $PWD entry right above in the docs.